packages/expo-blur/src/BlurView.tsx

```diff
@@ -29,15 +23,15 @@ class BlurView extends React.Component<BlurViewProps & BlurViewForwardedRefProp>
       blurReductionFactor = 4,
       style,
       children,
-      forwardedRef,
       ...props
     } = this.props;
     return (
       <View {...props} style={[styles.container, style]}>
         <NativeBlurView
-          ref={forwardedRef}
+          ref={(component) => {
+            this.blurViewRef = component;
+          }}
```

Comment:
This should work too 🤔
```suggestion
          ref={this.blurViewRef}
```